components/ws-manager-bridge/src/bridge.ts.rej

```diff
@@ -0,0 +1,11 @@
+--- components/ws-manager-bridge/src/bridge.ts
```

Comment:
Are the .orig and .rej added on purpose?

Reply:
Those seem to be merge left-overs. Please remove, the rest LGTM!

Reply:
Removed